### ISSUE TITLE
No source code license

### ISSUE THREAD

**Issue by csolisr** [gsteph/groove-dl#15]
Currently, Groove-DL has no license, which means it's fully copyrighted. Have you considered adding a license (like the GPL or the BSD-type)?

**Comment by gsteph** [comment]
I have. But I never bothered to lookup how to GPL it. If you want to do
something with the source code just ask me.

**Comment by csolisr** [comment]
To use the GPL, you can read this manual: https://www.gnu.org/licenses/gpl-howto.html

**Comment by yasoob** [comment]
I would like to merge this into youtube-dl. Can you release the source code into Public Domain ? That is because youtube-dl is also released to Public Domain. the link for the project is http://www.github.com/rg3/youtube-dl

**Comment by gsteph** [comment]
Public Domain is not exactly suitable for me.
Why does it have to be like youtube-dl?

**Comment by yasoob** [comment]
There is not any need for changing the license. I was just asking because
if you release it into the public domain we can integrate it into
youtube-dl and then your code can be helpful to a whole lot of other people
as well.

**Comment by gsteph** [comment]
Yes I understand. I meant public domain itself sounds a bit scary. Would the (L)GPL work for you?

**Comment by yasoob** [comment]
Hmm no. Sorry for that. We can only merge code which is released to the public domain.

**Comment by gsteph** [comment]
Isn't your software going to be open source? If not, I'm afraid I can't license you.

**Comment by yasoob** [comment]
Our project is open source and has always been. Public Domain is a truly Open Source license. Read this extract taken from http://creativecommons.org/licenses/publicdomain/

> Dedicator recognizes that, once placed in the public domain, the Work may be freely reproduced, distributed, transmitted, used, modified, built upon, or otherwise exploited by anyone for any purpose, commercial or non-commercial, and in any way, including by methods that have not yet been invented or conceived.

**Comment by gsteph** [comment]
I'm aware. I'd just have an issue with somebody "exploiting" my code in any way. I don't want it that open.
How come you can't combine the code if it's under the GPL?

**Comment by yasoob** [comment]
We want a consistent license. We can not license one part and keep the
other unlicensed. However even if I agree the other contributors won't
agree.

**Comment by gsteph** [comment]
I really don't quite get the issue. You'd just mention that certain files of yours contain code from the groove-dl project and supply the LICENSE file. Isn't that all?

**Comment by yasoob** [comment]
We don't want to do that. We want to keep the source code under a single license.
